src/app/CommandSender.cpp

```diff
@@ -82,6 +82,29 @@ CHIP_ERROR CommandSender::SendCommandRequest(const SessionHandle & session, Syst
     return SendInvokeRequest();
 }
 
+CHIP_ERROR CommandSender::SendGroupCommandRequest(const SessionHandle & session, System::Clock::Timeout timeout)
```

Comment:
Should error out from `SendCommandRequest` if that's handed a group session, and error out from here for a non-group session?

Reply:
Added the check.